.gitignore

```diff
@@ -164,6 +164,9 @@ cython_debug/
 Backend/src/data/**/*.csv
 !Backend/src/data/.gitkeep
 
+Backend/src/data/info/**/.info
```

Comment:
This does not seem to work, the file is still there. I think you might mean to write `Backend/src/data/info/**/*.info`

Reply:
done

Reply:
done